cypress/integration/business-pages/business-pages.test.js

```diff
@@ -9,20 +20,27 @@ describe('Business pages test ', () => {
   let enText;
 
   before(() => {
-    pageCode = 'page';
-    uaHeader = 'Сторінка';
-    enHeader = 'Page';
-    uaText = 'деякий текст';
-    enText = 'some text';
+    pageCode = addedBusinessPage.code;
+    uaHeader = addedBusinessPage.title[0].value;
+    enHeader = addedBusinessPage.title[1].value;
+    uaText = addedBusinessPage.text[0].value;
+    enText = addedBusinessPage.text[1].value;
   });
```

Comment:
Maybe, it would be better to move import this from variables with more suitable name?

Reply:
fixed